kitchen/ph5tostationxml.py

```diff
@@ -401,7 +401,7 @@ def create_obs_channel(self, station_list, deployment, cha_code, loc_code,
         if self.ph5.Das_t.has_key(das):
             Das_t = ph5API.filter_das_t(self.ph5.Das_t[das]['rows'],
                                         station_list[deployment][0]['channel_number_i'])
-            Receiver_t=self.ph5.get_receiver_t (Das_t[0], by_n_i=True)
+            Receiver_t=self.ph5.get_receiver_t (Das_t, by_n_i=True)

```

Comment:
It might have been part of it. I'm still working on that.  The full fix for it may be to give a start and end time when reading the das table in get_response_inv as well. I'm testing this now and will let you know. I will also need to fix PH5API so that 'rows' always gets set. 16-013 is putting it in a strange state and I'll let you know when i figure out why.